documentation/staging/1.2/content/navigate/model.md

```diff
@@ -31,9 +31,17 @@ and [Archive File](https://oracle.github.io/weblogic-deploy-tooling/concepts/arc
 Tooling documentation.
 
 ### Design View
-In the current release, the `Design View` is empty.  Our intention is to eventually provide a WebLogic
-console-like set of pages that will make it easier to create or edit a model using an interface more familiar to
-WebLogic administrators and developers.
+The `Design View` is provided through integration with the [WebLogic Remote Console](https://github.com/oracle/weblogic-remote-console/releases).
+For version details, see [Prerequisites]({{< relref "/setup/prerequisites.md" >}}).
+
+Upon first access, provide the location of the Remote Console installation directory, then click **Start WebLogic Remote Console**.
+Then use the WebLogic Server domain configuration pages to create or edit WDT model files. For reference, see
```

Comment:
I wonder is calling them the "Model Design View pages" instead of the "WebLogic Server domain configuration pages" might make more sense.

Reply:
Yes, I'm ok with that change.